app/_data/tables/install_options_konnect.yml

```diff
@@ -5,33 +5,27 @@ features:
   - name: "Docker"
     items:
       - name: "Docker on MacOS"
-        url: /konnect/runtime-manager/runtime-instances/gateway-runtime-docker/
         ga: true
         icon: /assets/images/icons/documentation/macos-logo.png
-      - name: "Docker on Windows"
-        url: /konnect/runtime-manager/runtime-instances/gateway-runtime-docker/
-        ga: true
+      - name: "Windows"
         icon: /assets/images/icons/documentation/windows-logo.svg
-      - name: "Docker on Linux"
-        url: /konnect/runtime-manager/runtime-instances/gateway-runtime-docker/
+        ga: true
+      - name: "Linux (Docker)"
         ga: true
         icon: /assets/images/icons/documentation/docker.png
-      - name: "Custom Docker configuration"
-        url: /konnect/runtime-manager/runtime-instances/gateway-runtime-docker/
+      - name: "Advanced Docker configuration"
```

Comment:
Since this one will have a page, I think we should add the link to this one.

Reply:
I also didn't see the page in the nav, but I think we were planning to add that one (and just that one) back?